soepy/python/simulate/simulate_auxiliary.py

```diff
@@ -18,36 +18,33 @@ def pyth_simulate(attr_dict, state_space_args, periods_emax):
         state_space_args[3],
     )
 
-    # Unpack parameter from the model specification
-    educ_min = attr_dict["INITIAL_CONDITIONS"]["educ_min"]
-    educ_max = attr_dict["INITIAL_CONDITIONS"]["educ_max"]
-    num_periods = attr_dict["GENERAL"]["num_periods"]
-    num_agents_sim = attr_dict["SIMULATION"]["num_agents_sim"]
-    seed_sim = attr_dict["SIMULATION"]["seed_sim"]
-    shocks_cov = attr_dict["DERIVED_ATTR"]["shocks_cov"]
-    optim_paras = attr_dict["PARAMETERS"]["optim_paras"]
-    delta = attr_dict["CONSTANTS"]["delta"]
-
-    educ_years = list(range(educ_min, educ_max + 1))
-    np.random.seed(seed_sim)
-    educ_years = np.random.choice(educ_years, num_agents_sim)
+    educ_years = list(range(model_params.educ_min, model_params.educ_max + 1))
+    np.random.seed(model_params.seed_sim)
```

Comment:
We need to revisit the setting of the random seed. This appears do be done repeatedly, for example in `draw_disturbances` again.

Reply:
The seed in the code is set twice. There is one seed value for the generation of initial conditions during the simulation, and an additional one for the max calculation. I thought that this is in order.

Reply:
Thanks, yes this is fine.